Source/swiftlint/Commands/Lint.swift

```diff
@@ -0,0 +1,54 @@
+import ArgumentParser
+
+extension SwiftLint {
+    struct Lint: ParsableCommand {
+        static let configuration = CommandConfiguration(abstract: "Print lint warnings and errors")
+
+        @OptionGroup
+        var common: LintOrAnalyzeArguments
+        @Option(help: pathOptionDescription(for: .lint))
+        var path: String?
+        @Flag(help: "Lint standard input.")
+        var useSTDIN = false
+        @Flag(help: quietOptionDescription(for: .lint))
+        var quiet = false
+        @Option(help: "The directory of the cache used when linting.")
+        var cachePath: String?
+        @Flag(help: "Ignore cache when linting.")
+        var noCache = false
+        @Flag(help: "Run all rules, even opt-in and disabled ones, ignoring `only_rules`.")
+        var enableAllRules = false
+        @Argument(help: pathsArgumentDescription(for: .lint))
+        var paths = [String]()
+
+        mutating func run() throws {
+            let options = LintOrAnalyzeOptions(
+                mode: .lint,
+                paths: paths + [path ?? ""],
```

Comment:
looks like this logic differs a bit from before?

```swift
            let allPaths: [String]
            if !path.isEmpty {
                allPaths = [path]
            } else {
                allPaths = paths
            }
```

wouldn't this mean if you passed explicit paths you would also get all files from PWD as well as them?

Reply:
Good catch. Previously `path` defaulted to `""` which is treated as PWD. So without this, not specifying `--path` or `--paths` means that nothing gets linted. So I need to handle that case now, but I'll fix this bug.

Reply:
Should be fixed in 9adb2dea.